src/platform/drivers/mss_can/mss_can.c

```diff
@@ -801,7 +801,7 @@ MSS_CAN_send_message
             this_can->hw_reg->TxMsg[mailbox_number].TXB.L = (pmsg->L | \
                                                    CAN_TX_WPNH_EBL | \
                                                    CAN_TX_REQ);
-            success = CAN_VALID_MSG;
+            success = CAN_OK;

```

Comment:
Looked at the API description and the proposed change does not seem appropriate.
Also realized that probably there is a confusion because some of the APIs return zero value (CAN_OK) to indicate success whereas other APIs including MSS_CAN_send_message() returns a non-zero value (CAN_VALID_MSG) as success. However, the values are documented.
We will try to reduce the confusion as much as possible when we make update to the driver next.